dap-etl/transformation/src/main/scala/org/broadinstitute/monster/dap/afus/AfusTransformationPipelineBuilder.scala

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.monster.dap.afus
+
+import com.spotify.scio.ScioContext
+import com.spotify.scio.values.SCollection
+import org.broadinstitute.monster.common.{PipelineBuilder, StorageIO}
+import org.broadinstitute.monster.dap.common.{Args, RawRecord, TransformationHelper}
+import org.slf4j.{Logger, LoggerFactory}
+
+object AfusTransformationPipelineBuilder extends PipelineBuilder[Args] {
+  /**
+    * Schedule all the steps for the Dog Aging transformation in the given pipeline context.
+    *
+    * Scheduled steps are launched against the context's runner when the `run()` method
+    * is called on it.
+    *
+    * Adding implicit logger so we can associate it with the PipelineBuilder object
+    */
+  implicit val logger: Logger = LoggerFactory.getLogger(getClass)
+
+  override def buildPipeline(ctx: ScioContext, args: Args): Unit = {
+    val rawRecords = readRecords(ctx, args)
+
+    val owners =
+      rawRecords.transform("Map AFUS Owners")(_.flatMap(AfusOwnerTransformations.mapAfusOwnerData))
+
+    StorageIO.writeJsonLists(owners, "AFUS Owners", s"${args.outputPrefix}/afus_owner")
+    ()
+  }
+
+  /** Read in records and group by study Id, with field name subgroups. */
+  def readRecords(ctx: ScioContext, args: Args): SCollection[RawRecord] = {
+    TransformationHelper.readRecordsGroupByStudyId(ctx, args.inputPrefix)
```

Comment:
🤔 thinking this through a bit -- will this smoosh all records with the same study ID into a single raw record potentially losing data for multiple years? How will this work for multiple years and is that what we want?

 i.e,. in the fixture data for the related test what is the behavior when this is called on it?

```
{"record":"11111","redcap_event_name":"feb2020_arm_1","field_name":"ss_num_people_hh","value":"2"}
{"record":"11111","redcap_event_name":"feb2021_arm_1","field_name":"ss_num_people_hh","value":"4"}
```